platform/linux-generic/odp_hash_crc32.c

```diff
@@ -9,6 +10,23 @@
 #include <odp/api/hash.h>
 #include <odp/api/align.h>
 
+#include <odp/api/abi/hash.h>
+
+#ifdef _ODP_CRC32_EXTENSIONS
+/*
+ * Use gcc builtin CRC32 intrinsics for hash calculation.
```

Comment:
But in this case we're actually using ACLE defined functions. So maybe "Use architecture-specific extensions" or something like that. The same goes for the commit comment.

Reply:
I will make the change. Thanks!